src/node_api_helpers.h

```diff
@@ -38,6 +45,289 @@
   NAPI_DISALLOW_MOVE(CLASS)
 
 namespace Napi {
+  inline napi_value Value() {
+    napi_env env = napi_get_current_env();
+    return napi_new_value(env);
+  }
+
+
+  // Napi::New helpers
+  inline napi_value New() {
+    return napi_get_null(napi_get_current_env());
+  }
+  inline napi_value New(bool val) {
```

Comment:
Unfortunately most of these helper functions are going to have to change significantly to accommodate the new error-handling pattern. You might want to wait until that change goes in so you can build the helpers on it.